modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -74,13 +74,13 @@ func (suite *KeeperTestSuite) TestEscrowPacketFee() {
 
 			tc.malleate()
 			fee := types.Fee{ackFee, receiveFee, timeoutFee}
-			identifiedPacketFee := &types.IdentifiedPacketFee{PacketId: packetId, Fee: fee, Relayers: []string{}}
+			identifiedPacketFee := &types.IdentifiedPacketFee{PacketId: packetId, Fee: fee, RefundAddress: refundAcc.String(), Relayers: []string{}}
```

Comment:
Nit: Type instantiation is almost always easier to read when written across multiple lines:

```
identifiedPacketFee := &types.IdentifiedPacketFee{
    PacketId: packetID, 
    Fee: fee, 
    RefundAddress: refundAcc.String(), 
    Relayers: []string{},
}
```

I know it was like this previously, we can fix these in the audit! :))

Reply:
Yea prefer fixing this later. i want to merge soon, so we can unblock me and @charleenfei